frontend/src/components/LoginForm.vue

```diff
@@ -0,0 +1,84 @@
+//TODO: Add basic input validation
+//FIXME: Not entering anything in the input fields throws errors in the console
+
+
+<script setup lang="ts">
+import { Form, FormField, type FormSubmitEvent } from '@primevue/forms';
+import { InputText, Message, Button } from 'primevue';
+import { useRouter } from 'vue-router';
+import { useToast } from 'primevue/usetoast';
+import Toast from 'primevue/toast';
+const router = useRouter();
+const toast = useToast();
+const onFormSubmit = async (event: FormSubmitEvent) => {
+    let username = event.states.username.value;
+    let password = event.states.password.value;
+    await login(username, password, router, toast);
+};
+const getSalt = async (username: string) => {
+    let response = await fetch(`http://localhost:2024/login/${username}`);
+    let data = await response.json();
+    return data.passwordSalt;
+};
```

Comment:
Here you don't check for the response code. If it is not 200, there won't be a "passwordSalt" field. It's ok for prototyping, but it should be changed in the future.

Reply:
Referenced in issue #42